src/tag.rs

```diff
@@ -117,7 +117,7 @@ impl<'a> Tag {
     /// Removes an ID3v2 tag from the specified file.
     ///
     /// Returns true if the file initially contained a tag.
-    pub fn remove_from_file(mut file: &mut fs::File) -> crate::Result<bool> {
+    pub fn remove_from_file(mut file: &mut impl StorageFile) -> crate::Result<bool> {
```

Comment:
StorageFile is implemented for &mut impl StorageFile itself, so we do not need the &mut ref in the type. Could you change all instances to just be impl StorageFile?

The member functions of `Tag` are part of the public API, but given that `fs::File` implements `StorageFile` this should not cause any breakages :grimacing:

Reply:
Good point, I didn't notice! Sorry about that. I've just amended the commit with your suggestion. I can confirm tests continue to run, and my application continues to work as expected!

Reply:
Nice, thanks!